overrides/ios-override.json

```diff
@@ -104,6 +104,24 @@
                 ]
             }
         },
+        "trackerAllowlist": {
+            "state": "enabled",
+            "settings": {
+                "allowlistedTrackers": {
+                    "monetate.net": {
+                        "rules": [
+                            {
+                                "rule": "monetate.net",
+                                "domains": [
+                                    "kleen-ritecorp.com"
+                                ],
+                                "reason": "blank page, and product carousels not working - https://app.asana.com/0/1200277586140538/1205303551800514/f"
```

Comment:
text descriptions are now deprecapted we usually add an issue link with the description of the issue. I made one here https://github.com/duckduckgo/privacy-configuration/issues/1231